package.json

```diff
@@ -1,46 +1,100 @@
 {
   "name": "@xarc/module-dev",
   "version": "1.0.0",
-  "main": "lib/arch-clap.js",
-  "description": "Archetype for NodeJS Module (Development)",
-  "scripts": {},
+  "main": "index.js",
+  "types": "dist/arch-clap.d.ts",
+  "description": "Support for developing node.js modules with mocha/eslint/typescript",
+  "scripts": {
+    "build": "tsc",
+    "test": "clap -n xarc/test",
+    "compile": "tsc",
+    "prepublishOnly": "tsc",
+    "postinstall": "node install.js"
+  },
   "repository": {
     "type": "git",
     "url": "https://github.com/jchip/module-dev.git"
   },
   "author": "Joel Chen",
   "license": "Apache-2.0",
   "files": [
+    "install.js",
     "config",
-    "lib"
+    "dist"
+  ],
+  "keywords": [
+    "node",
+    "node.js",
+    "npm",
+    "module",
+    "develop",
+    "mocha",
+    "eslint",
+    "typescript"
   ],
   "dependencies": {
-    "babel-eslint": "^10.0.1",
     "chai": "^4.2.0",
-    "eslint": "^5.15.0",
-    "eslint-config-walmart": "^2.2.1",
-    "eslint-plugin-filenames": "^1.1.0",
-    "mocha": "^6.0.2",
-    "nyc": "^13.3.0",
+    "filter-scan-dir": "^1.0.10",
+    "lodash": "^4.17.15",
+    "mocha": "^7.1.0",
+    "nyc": "^15.0.0",
     "sinon": "^7.2.6",
     "sinon-chai": "^3.3.0",
-    "xclap": "^0.2.27",
+    "xclap": "^0.2.43",
```

Comment:
I'm assuming this will be published later?

Reply:
it's published :)